app.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express')
```

Comment:
without the test this is pointless. How to we know it works?
start with the tests which define your API behaviour, then provide minimal implementation which makes the tests pass.